apache2/mod_security2.c

```diff
@@ -692,9 +692,8 @@ static int hook_post_config(apr_pool_t *mp, apr_pool_t *mp_log, apr_pool_t *mp_t
         first_time = 1;
         apr_pool_userdata_set((const void *)1, "modsecurity-init-flag",
                 apr_pool_cleanup_null, s->process->pool);
-    } else {
-        modsecurity_init(modsecurity, mp);
     }
+    modsecurity_init(modsecurity, mp);
```

Comment:
Hum.. I've got the change to use the tmp-file-names, but what about this change?

If i recall correctly this will be executed twice: while checking the configuration fro syntax error and while effective loading it. We may want to init ModSec only at the second one. Right?

Reply:
I was not sure about that. What about moving it then just after "first_time = 1;" ? Then it will really only be called once at the beginning.

Reply:
Not sure, we may not need to change it at all. I've created this branch **v2/dev/pull_1957**. It contains all the modifications except this one.